front/se_project/src/Pages/admin.jsx

```diff
@@ -15,19 +17,58 @@ class Admin extends Component {
                     <div className='container w-75 p-3 shadow bg-white rounded mb-5'>
-                            <ShowArtworkAdCost/>
+                            <ArtworkAdAdditionalInfo  showImages showText showCostText/>
                             <div className='d-flex' style={{marginRight:'2%'}}>
                                 <div className='ms-2'>
-                                    <Button width= 'auto' height= 'auto' text= 'تایید' fontSize='1.17vw'/*onClick*//>
+                                    <Button width= 'auto' height= 'auto' text= 'تایید' fontSize='1.17vw'/*onClick/>
                                 </div>
-                                <Button width= 'auto' height= 'auto' text= 'حذف' fontSize='1.17vw'/*onClick*//>
+                                <Button width= 'auto' height= 'auto' text= 'حذف' fontSize='1.17vw'/*onClick/>
+
+ class Admin extends Component {
+//     state = {  } 
+//     render() { 
+//         return (
+//             <div>
+//                 <Navbar/>
+//                 <Breadcrumbb is1OrNot='true' first='حساب کاربری'/>
+//                 <div className='d-flex flex-column min-vh-100'>
+//                     <HorizontalLine value='آگهی‌های جدید'/>
+//                     <div className='container w-75 p-3 shadow bg-white rounded mb-5'>
+//                             <ArtworkAdAdditionalInfo  showImages showText showCostText/>
+//                             <div className='d-flex' style={{marginRight:'2%'}}>
+//                                 <div className='ms-2'>
+//                                     <Button width= 'auto' height= 'auto' text= 'تایید' fontSize='1.17vw'/*onClick>*/
+//                                 </div>
+//                                 <Button width= 'auto' height= 'auto' text= 'حذف' fontSize='1.17vw'/*onClick*//>
+//                             </div>
+//                     </div>
+//                 </div>
+//             </div>
+//         );
+//     }
+// }
+ 
+// export default Admin;*/
+const Admin = (props) => {
+    return ( 
+        <div>
+            <Navbar/>
+            <Breadcrumbb is1OrNot='true' first='حساب کاربری'/>
+            <div className='d-flex flex-column min-vh-100'>
+                <HorizontalLine value='آگهی‌های جدید'/>
+                <div className='container w-75 p-3 shadow bg-white rounded mb-5'>
+                        <ShowArtworkAdCost />
+                        <div className='d-flex' style={{marginRight:'2%'}}>
+                            <div className='ms-2'>
+                                <Button width= 'auto' height= 'auto' text= 'تایید' fontSize='1.17vw'/*onClick*//>
                             </div>
-                    </div>
+                            <Button width= 'auto' height= 'auto' text= 'حذف' fontSize='1.17vw'/*onClick*//>
+                        </div>
                 </div>
                 <Footer/>
             </div>
-        );
-    }
+        </div>
+     );
 }
  
 export default Admin;
```

Comment:
I think we can delete comments related to class based component.

Reply:
I check it